src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -452,57 +418,24 @@ private function autoPrependText(): void
         }
     }
 
-    private function writeBuffer(string $message, bool $newLine, int $type): void
+    private function createBlock(iterable $messages, string $type = null, string $style = null, string $prefix = ' ', bool $padding = false, bool $escape = false): array
     {
-        // We need to know if the last chars are PHP_EOL
-        $this->bufferedOutput->write($message, $newLine, $type);
+        return (new OutputBlock())
+            ->setMessages($messages)
+            ->setFormatter($this->getFormatter())
+            ->setLineLength($this->lineLength)
+            ->setType($type)
+            ->setStyle($style)
+            ->setPrefix($prefix)
+            ->setPadding($padding && $this->isDecorated())
+            ->setEscape($escape)
+            ->getLines();
     }
 
-    private function createBlock(iterable $messages, string $type = null, string $style = null, string $prefix = ' ', bool $padding = false, bool $escape = false): array
+    protected function getErrorOutput()
```

Comment:
protected methods should come before private ones btw.

Reply:
ok thanks